contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -1758,13 +1758,14 @@ public EnumKey(HKEY hKey, int dwIndex) {
 	 * @return A environment block
 	 */
 	public static String getEnvironmentBlock(Map<String, String> environment) {
-		StringBuilder out = new StringBuilder();
+		StringBuilder out = new StringBuilder(environment.size() * 32 /* some guess about average name=value length*/);
 		for (Entry<String, String> entry : environment.entrySet()) {
-			if (entry.getValue() != null) {
-				out.append(entry.getKey() + "=" + entry.getValue() + "\0");
+		    String    key=entry.getKey(), value=entry.getValue();
```

Comment:
This has been the style of changes I submitted in the past and no one remarked about it. I have taken notice and will make an effort to have future submissions adhere to this style. As far as **this** submission goes, I would ask that you accept this change as-is, since back-formatting all the code I wrote with the spaces around the assignment operator would be tedious (I know Eclipse has a formatting option but I am afraid that other formatting issues will appear that may not be the way you like it - unless you can send me the **exact** configuration file for the Eclipse style).